e2e/server/index.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable @lwc/lwc/no-async-await */
```

Comment:
I think we prefer promises, no `async`

Reply:
Hey Andrew. I refactored almost everything to the plain promise API with .then()/.catch() …

… except for the `testRunner.js`. This is the main script file running the tests. I started rewriting it (c623ffe0273814ff4337b9a8b37870c7a7bd3503), but it just turned out to become super messy with the .then() API, like the readability went down drastically.
This is not part of the app code but is a separate script. If we are fine with that I'd like to keep that script using `async`, as it really feels more like a procedural script, so reading it with `await` is just more simple.

If that's not fine the refactor will take a bit more time because I don't want that script file to become incomprehensible 😄

Reply:
I _think_ we've basically "outlawed" async/await in the whole repo (including a bunch of JS code in `.github/actions`) and adding it would not be great as we specifically decided against it in our style guide: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#esnext-are-we-allowed-to-use-insert-new-language-feature-why-or-why-not

Apologies for the extra work here. 

One extra note: I think it _would_ be OK to use `async/await` if it was an npm package, but not in the source.